state/pgstatestorage.go

```diff
@@ -2697,6 +2697,34 @@ func (p *PostgresStorage) GetDSL2Transactions(ctx context.Context, minL2Block, m
 	return l2Txs, nil
 }
 
+// GetNativeBlockHashesInRange return the state root for the blocks in range
+func (p *PostgresStorage) GetNativeBlockHashesInRange(ctx context.Context, fromBlock, toBlock uint64, dbTx pgx.Tx) ([]common.Hash, error) {
+	const l2TxSQL = `
+    SELECT l2b.state_root
+      FROM state.l2block l2b
+     WHERE l2_block_num BETWEEN $1 AND $2
```

Comment:
The name of the field is not l2_block_num is block_num

Reply:
good catch, I missed the unit test for this method. I'll add it.